scripts/ldev2pcs

```diff
@@ -20,6 +20,29 @@ def should_skip(line):
         return True
     return False
 
+def get_pacemaker_major_number():
+    '''Get the rpm major number of pacemaker.'''
+    rpm_version = subprocess.check_output(
+        ['rpm', '-q', 'pacemaker']
+    ).strip()
+
+    captured_major_number = re.match(r'pacemaker-(?P<major>\d+).*$', rpm_version)
+
+    if captured_major_number is not None:
```

Comment:
What happens if captured_major_number _is_ None?

Reply:
It returns None, but doesn't try to cast None to an int. I can have it just fail if it can't reasonably return an int here.